federation/pkg/federation-controller/replicaset/replicasetcontroller.go

```diff
@@ -43,6 +43,7 @@ import (
 	"k8s.io/kubernetes/pkg/util/flowcontrol"
 	"k8s.io/kubernetes/pkg/util/wait"
 	"k8s.io/kubernetes/pkg/util/workqueue"
+	_ "k8s.io/kubernetes/pkg/util/workqueue/prometheus"
```

Comment:
I think you try to put this import in package main instead of in these libraries.